open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,33 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service that they would
+        that they would prefer response to be paginated. If pageSize is specified, the service should return exactly the number of items specified by it.
+        
+        Clients that support pagination initiate the request by sending an empty pageToken e.g. GET /tables?pageToken
+        Servers supporting pagination would recognize pagination is requested due to the presence of the pageToken
+        and honor the contracts specified above by returning a NextPageToken in response if there are more results available.
+        Servers that are non-pagination aware will ignore the token and return all results.
```

Comment:
I think we can follow the pageSize description below and separate this to "For servers that support pagination, ..." and "For servers that do not support pagination, ..."